test/kube_checks.go

```diff
@@ -73,3 +74,15 @@ func WaitForPodListState(client *KubeClient, inState func(p *corev1.PodList) (bo
 		return inState(p)
 	})
 }
+
+// GetConfigMap gets the knative serving config map.
+func GetConfigMap(client *KubeClient) k8styped.ConfigMapInterface {
+	return client.Kube.CoreV1().ConfigMaps("knative-serving")
```

Comment:
Is there a reason why is the namespace hardcoded?

And does this helper apply to anything beyond `knative-serving`?

Reply:
good point: https://github.com/knative/pkg/pull/237